common/src/main/java/io/sphere/sdk/models/LocalizedStrings.java

```diff
@@ -137,24 +137,10 @@ public String toString() {
 
     @SuppressWarnings("unused")//used by Jackson JSON mapper
     @JsonAnySetter
-    private void set(final String languageTag, String value) {
+    private void set(final String languageTag, final String value) {
         translations.put(Locale.forLanguageTag(languageTag), value);
     }
 
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-        LocalizedStrings localized = (LocalizedStrings) o;
-        return this.translations.equals(localized.translations);
-    }
-
-    @Override
-    public int hashCode() {
-        return translations.hashCode();
-    }
-
```

Comment:
The class was mostly from the Play SDK, in the JVM SDK we just extend Base and do not reimplement equals, hashCode and toString.